test/runtests.sh

```diff
@@ -19,6 +19,20 @@ if [[ $build_type != "-d" && $build_type != "-t" ]]; then
     elif [[ -f "$test_path/../BuildLinux/Test/ch" ]]; then
         echo "Warning: Test build was found"
         build_type="-t"
+    elif [[ -f "$test_path/../BuildLinux/Release/ch" ]]; then
+        # TEST flags are not enabled for release build
+        # however we would like to test if the compiled binary
+        # works or not
+        CH="$test_path/../BuildLinux/Release/ch"
+        echo "Warning: Release build was found"
```

Comment:
I think in all cases, warning is a bit strong since there's no case where you won't get a warning. Maybe just change these to "Note: using Release build" etc.